evaluation/app_chat_target.py

```diff
@@ -0,0 +1,111 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import logging
+import os
+from pathlib import Path
+
+from dotenv import load_dotenv
+from pyrit.chat_message_normalizer import ChatMessageNop, ChatMessageNormalizer
+from pyrit.common import default_values, net_utility
+from pyrit.memory import MemoryInterface
+from pyrit.models import (
+    ChatMessage,
+    PromptRequestResponse,
+    construct_response_from_request,
+)
+from pyrit.prompt_target import PromptChatTarget
+
+from evaluation.utils import load_config
+
+load_dotenv()
+
+EVALUATION_DIR = Path(__file__).parent
+
+logger = logging.getLogger(__name__)
+
+
+class AppChatTarget(PromptChatTarget):
+
+    BACKEND_URI: str = os.environ.get("BACKEND_URI", "").rstrip("/") + "/ask"
+
+    def __init__(
+        self,
+        *,
+        endpoint_uri: str = None,
+        chat_message_normalizer: ChatMessageNormalizer = ChatMessageNop(),
```

Comment:
Note that we're not using this. We're not passing a normaliser on initialisation currently and the default one simply returns the messages. I'm not saying that we should remove it, just raising it FYI.